BaseFigure.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FiguresLR1
+{
+    internal abstract class BaseFigure //current build
```

Comment:
what is the plot of this comment?

Reply:
There is no plot. I've just forgotten to delete it 😅